Bluedot-Innovation/PointSDK-BrazeIntegrationExample-Android
# Po/bd 2357 simple init

To test:

1. update `projectId` in **MainApplication.kt**
2. update `com_appboy_api_key` in **braze.xml**
3. udpade `com_appboy_firebase_cloud_messaging_sender_id` in **braze.xml**

trigger an entry. Observe event is coming to both Braze Portal and Push Notification.
